From: Remi Collet <remi@remirepo.net>
Date: Wed, 10 Apr 2024 08:59:32 +0200
Subject: NEWS

(cherry picked from commit 366cc249b7d54707572beb7096e8f6c65ee79719)
(cherry picked from commit dcdd49ef3bfbd8ccc778850d6a0f9b98adf625d4)
(cherry picked from commit 8642473b624f809b768180b104c013f74e3a99a0)
(cherry picked from commit ee591001f7a3db7405b4fa027659768c2355df6d)
(cherry picked from commit 035bc48bafe5d567f4ab8de6d1752a724e361690)
---
 NEWS | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/NEWS b/NEWS
index c82aea8..216d8ee 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,12 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 8.1.28
+
+- Standard:
+  . Fixed bug GHSA-wpj3-hf5j-x4v4 (__Host-/__Secure- cookie bypass due to
+    partial CVE-2022-31629 fix). (CVE-2024-2756) (nielsdos)
+
 Backported from 8.0.30
 
 - Libxml:
